web/src/features/map/Map.tsx

```diff
@@ -145,6 +145,10 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
     isSourceLoaded,
     spatialAggregate,
     isSuccess,
+    isLoading,
+    isError,
+    worldGeometries.features,
+    theme.clickableFill,
```

Comment:
Because all of these are in one way or another used inside the hook, that means the outcome is dependant on their value and if they change the hook should rerun. They probably won't change though but to react they *can* change which is why it wants them here.